app/server/ruby/lib/sonicpi/lang/core.rb

```diff
@@ -143,7 +143,31 @@ def __cueset(k, val, prefix)
       def set(k, val)
         __cueset(k, val, "set")
       end
+      doc name:           :set,
+          introduced:     Version.new(3,0,0),
+          summary:        "Store information in the Time State",
+          doc:            "Store information in the Time State set for the current time for either the current or any other thread. If called multiple times without an intervening call to `sleep`, `sync`, `set` or `cue`, the last value set will prevail.
```

Comment:
I think this would make slightly more sense like this:
```suggestion
          doc:            "Store information in the Time State for the current time for either the current or any other thread. If called multiple times without an intervening call to `sleep`, `sync`, `set` or `cue`, the last value set will prevail.
```